src/pyOpenMS/pxds/OpenSwathDataStructures.pxd

```diff
@@ -3,21 +3,36 @@ from smart_ptr cimport shared_ptr
 
 cdef extern from "<OpenMS/OPENSWATHALGO/DATAACCESS/DataStructures.h>" namespace "OpenSwath":
 
+  # See ../addons/OSBinaryDataArray.pyx
   cdef cppclass OSBinaryDataArray:
         OSBinaryDataArray() nogil except +
         OSBinaryDataArray(OSBinaryDataArray &) nogil except + # compiler
         libcpp_vector[double] data
         libcpp_string description
+
+        libcpp_vector[double] getData(self) #wrap-ignore wrap-doc:Access to a copy of the underlying data using a numpy array
+        libcpp_vector[double] getData_mv(self) #wrap-ignore wrap-doc:Access to the underlying data using a memory view
```

Comment:
Hmm now I am actually not 100% sure if this will generate the docs. I would think they need to be in the pyx as docstrings. What do you think? I guess having them in both locations would be safe but also redundant. Docs could also be extended for such a "dangerous" function.

Reply:
turns it out is not showing up in the docs (makes sense since we add wrap-ignore)